src/popper/runner_host.py

```diff
@@ -115,6 +116,10 @@ def __init__(self, init_docker_client=True, **kw):
         if not init_docker_client:
             return
 
+        docker_executables = ["docker"]
```

Comment:
we can skip checking for the `docker` binary. This is because in the case of Docker, the code doesn't strictly depend on the `docker` client binary. The docker-py library can directly talk to the daemon, which is implicitly done in line 125 (when `self._d.version()` is executed)

Reply:
Sounds good, makes sense! I'll make the change